docs/running-rules.md

```diff
@@ -34,7 +34,10 @@ semgrep --config=<ruleset-id> path/to/src
 semgrep --config=<ruleset-id> --config=path/to/yml path/to/src
 ```
 
-When the Registry is used, [usage metrics](../metrics) are collected.
+:::note
+* `--config auto` sends your repository's git URL to [Semgrep Registry](https://semgrep.dev/r) to find rules configured for your repository and as a key for cached rule recommendations.
+* When Semgrep Registry is used, [usage metrics](../metrics) are collected.
```

Comment:
Should this be “…are collected by default” or otherwise indicating that it happens unless the user operates Semgrep in a way not to collect those metrics?